src/projects/models.py

```
@@ -1,9 +1,18 @@
 from django.db import models
 from django.contrib.auth.models import User
 from django.utils import timezone
-from django.conf import settings
+from projects.tasks.fetch_favicon import fetch_favicon
```

Ruff

`projects.tasks.fetch_favicon.fetch_favicon` imported but unused

Remove unused import: `projects.tasks.fetch_favicon.fetch_favicon`

(F401)